src/ontology/uberon-edit.obo

```diff
@@ -209360,7 +209348,7 @@ property_value: provenance_notes "This class was sourced from an external ontolo
 id: UBERON:3000809
 name: accessory articulation
 def: "Paired, additional vertebral articulations located medial to the intervertebral space, adjacent to the vertebral centrum." [AAO:Cannatella_1985]
-is_a: UBERON:0000477 ! anatomical cluster
+is_a: UBERON:0034921 ! multi organ part structure
```

Comment:
Unsure if this refers to something that is "multi organ".

Reply:
Each vertebra is individually considered as a separate “organ” (a _vertebra_ is a _vertebral element_, which is a _skeletal element_, which is an _organ_), so I think it is consistent to say that an articulation between two vertebrae is a “multi organ part structure“ – it involves subparts of two different organs, which fits the definition of “multi organ part structure“.

But this makes me realise that this term can be classified even more precisely under _articulation_ (UBERON:0004905), which is a subclass of _multi organ part structure_. I’ll do that now.